en/docs/install-and-setup/upgrading-wso2-is-as-key-manager/upgrading-from-is-km-520-to-5100.md

```diff
@@ -12,58 +12,55 @@ The following information describes how to upgrade your **WSO2 API Manager (WSO2
 
     -   If you wish to upgrade your APIM environment from **API-M 2.0.0 to 3.1.0**, which is using the **internal** WSO2 Identity Server (**WSO2 IS**) **capabilities**, follow the instructions in [Upgrading API-M from 2.0.0 to 3.1.0](../upgrading-wso2-api-manager/upgrading-from-200-to-310.md).
 
-Upgrade WSO2 API-M **from WSO2 API-M 2.0.0 to 3.1.0** when using **WSO2 IS** as the **Key Manager:** as mentioned in below.
+When **WSO2 Identity Server (WSO2 IS)** is the **Key Manager**,
 
-Follow the instructions mentioned in [Upgrading API-M from 2.0.0 to 3.1.0](../upgrading-wso2-api-manager/upgrading-from-200-to-310.md), 
-but **instead of step 2 - (6)**, which explains how to migrate the WSO2 API-M Identity components, 
-follow the WSO2 Identity Server 5.10.0 documentation to migrate the WSO2 Identity Server (WSO2 IS) from version 5.2.0 to 5.10.0.
-[Migrate the WSO2 Identity Server (WSO2 IS) from version 5.2.0 to 5.10.0](https://is.docs.wso2.com/en/5.10.0/setup/migrating-to-5100/)
+Follow the instructions in [Upgrading API-M from 2.0.0 to 3.1.0](../upgrading-wso2-api-manager/upgrading-from-200-to-310.md),
+**instead of step 2 - (6)**, which explains how to migrate the WSO2 API-M Identity components, 
```

Comment:
Shouldn't the above sentence end at here? 
There are 3 sections in this sentence separated by commas and when we read as a whole the meaning is confusing. The below part should be a separate sentence AFAIU.

Reply:
This should be fixed in other pages too.